### PR TITLE
feat(v0): add basic desktop styling

### PR DESCRIPTION
**Pull request by kcvan**
feat(desktop): Add basic desktop styling for v0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
-# Bee Token Site
```
Comment (PakAndKap) [inline README.md]: Use this instead~

File: README.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+# bNest Token Site
 
 ### General
-- Email: thebeetoken@gmail.com
```
Comment (PakAndKap) [inline README.md]: Use this instead~

File: index.html [hunk 3]
```diff
@@ -1,41 +1,43 @@
 <!DOCTYPE HTML>
 <html lang="en">
 	<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1">
 		<meta charset="UTF-8">
 		<meta name="Referrer" content="origin">
-		<title>Bee Token</title>
```
Comment (PakAndKap) [inline index.html]: Use this instead~

File: index.html [hunk 4]
```diff
@@ -1,41 +1,43 @@
...
-		</div>
+		<section class="bn-main">
+      <div class="bn-main-container">
+          <img src="bee.jpg" id="bn-logo-enter">
+          <img src="bee.jpg" id="bn-logo-main">
+          <div id="bn-main-cta">
+            <form class="bn-main-cta-form" method="POST" action="http://formspree.io/thebeetoken@gmail.com">
+              <h1>The Bee Token - <span class="bn-main-cta-header-span">Reinventing Housing Exchange</span></h1>
+              <input class="bn-main-cta-email" type="email" name="email" placeholder="Your email">
+              <input class="bn-main-cta-hidden" type="hidden" name="_next" value="https://bnest-token.github.io/" />
```
Comment (PakAndKap) [inline index.html]: Change the value to "thebeetoken.github.io"

**Review by PakAndKap — CHANGES_REQUESTED** [review]
Few naming issues

**Comment by kcvan** [comment]
Requested Changes have been made to v0

**Pull request merged**